specifyweb/frontend/js_src/lib/components/WbPlanView/Mapper.tsx

```diff
@@ -480,7 +493,7 @@ export function Mapper(props: {
       <Ul
         aria-label={wbPlanText.mappings()}
         className={`
-          grid flex-1 auto-rows-max grid-cols-[theme(spacing.8)_max-content_auto]
+          grid flex-1 auto-rows-max grid-cols-[theme(spacing.24)_max-content_auto]
```

Comment:
(optional) refactor line 487 in this file to replace `&#8594;` with `{'→'}` or maybe even `→`